src/types.ts

```diff
@@ -28,7 +28,11 @@ type EnhancedRequestInit<T = string> = Omit<RequestInit, 'body' | 'method'> & {
   body?: JSONValue | BodyInit | null
   query?: SearchParams
   params?: PathParams<T>
-  trace?: (...args: Parameters<typeof fetch>) => void
+  trace?: (
+    fullUrl: string | URL,
+    init: EnhancedRequestInit,
+    response: Awaited<ReturnType<typeof fetch>>,
```

Comment:
Couldn't you use TypedResponse type here?

Reply:
sure, let me try